### DISCUSSION TITLE
Disable the notifications

### DISCUSSION THREAD

**Discussion by MDA-BlueByte** [Romanitho/Winget-AutoUpdate#461] [Q&A] [answered]
Hi

Does anybody know if it is possible to disable the "Toast" notifications when running the autoupdater?
![Screenshot_6](https://github.com/Romanitho/Winget-AutoUpdate/assets/149507698/2c68b274-6d96-4768-b20f-b08082d6ae1d)

**Comment by KnifMelti** [comment, marked as answer]
GPO/Reg/Install parameter:
_-NotificationLevel
Specify the Notification level: Full (Default, displays all notification), SuccessOnly (Only displays notification for success) or None (Does not show any popup)._
![image](https://github.com/Romanitho/Winget-AutoUpdate/assets/102996177/37aac0ec-1389-447d-b18f-ec88f445a2e6)
![image](https://github.com/Romanitho/Winget-AutoUpdate/assets/102996177/a45168a4-5d94-4efe-91d4-136014ee51e2)

**Reply by Romanitho** [reply]
It is explained in the readme.
You can also use WiGui to have a simplified way to install WAU (and apps):
https://github.com/Romanitho/Winget-Install-GUI as mentioned, NotificationLevel option to none

**Reply by MDA-BlueByte** [reply]
Perfect! Thank you!

I searched the readme, but with no luck.
I will try the Registry change, that will be the best solution for my case.

**Reply by Romanitho** [reply]
https://github.com/Romanitho/Winget-AutoUpdate#readme 😅
